### PR TITLE
Add format: pkcs12 to secrets

### PR DESCRIPTION
**Pull request by bogdan-kovalev**

### CODE REVIEW

File: samples/ra-vpn/saml-multi-region/saml-cert.yaml [hunk 1]
```diff
@@ -9,6 +9,7 @@ metadata:
   namespace: sfcn-system
 stringData:
   trustpoint: samlpkcs
+  format: pkcs12
```
Comment (jaredhancock31) [inline samples/ra-vpn/saml-multi-region/saml-cert.yaml:12]: Is this correct? We aren't importing this cert but also this isn't pkcs12 formatted right?

Comment (bogdan-kovalev) [reply samples/ra-vpn/saml-multi-region/saml-cert.yaml:12]: IDK, it says **samlpkcs** so I thought it's pkcs12.
I'll check

Comment (bogdan-kovalev) [reply samples/ra-vpn/saml-multi-region/saml-cert.yaml]: @jaredhancock31 , okay, addressed, and also made `sfcn-redis` consistent with other examples

Comment (bogdan-kovalev) [reply samples/ra-vpn/saml-multi-region/saml-cert.yaml]: @jaredhancock31 , that `sfcn-redis` is created by the helm chart. Reverted that change 😞

**Pull request merged**